docs/09-references/00-references.adoc

```diff
@@ -38,8 +38,9 @@
 
 // G
 - [[[gof,Gamma+94]]] Erich Gamma, Richard Helm, Ralph Johnson & John Vlissides. Design Patterns: Elements of Reusable Object-Oriented Software. Addison-Wesley. 1994.
+- [[[geewax,Geewax 2021]]] J. Geewax. API Design Patterns. Manning, 2021. This book lays out a set of design principles for building internal and public-facing APIs. 
 - [[[geirhos,Geirhos 2015]]] Matthias Geirhos. Entwurfsmuster: Das umfassende Handbuch (in German). Rheinwerk Computing Verlag. 2015
-- [[[gharbietal,Gharbi+2020]]] Mahbouba Gharbi, Arne Koschel, Andreas Rausch, Gernot Starke: Basiswissen Softwarearchitektur. 4. Auflage, dpunkt Verlag, Heidelberg 2020.
+- [[[gharbietal,Gharbi+2020]]] Mahbouba Gharbi, Arne Koschel, Andreas Rausch, Gernot Starke: Basiswissen Softwarearchitektur. 5. Auflage, dpunkt Verlag, Heidelberg 2024.
```

Comment:
Do we usually keep the reference Id permanent when a year changes or does it make sense to update the reference Id to 2024 to match the year of publication?
If we would add it as a new reference, it would probably be `gharbietal,Gharbi+2024`, so I think it would make sense to update the year now as well. But I can live with either way ;-)

Reply:
Discussed with @alxlo and decided to keep the year in the references. We update only every 24 month, and we consider it ok to update the references after such an intervall...

Therefore, we should update gharbietal to Gharbi+2024

Reply:
Good idea!

Reply:
resolved with commit 36f8b238c50ce9d201e09532a65356537671c37b